frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -246,12 +249,12 @@ export const sessionsTableLogic = kea<sessionsTableLogicType<SessionRecordingId>
                 ...overrides,
             }
 
-            return [router.values.location.pathname, params, router.values.hashParams]
+            return [router.values.location.pathname, params, router.values.hashParams, { replace }]
         }
 
         return {
-            setFilters: () => buildURL(),
-            loadSessions: () => buildURL(),
+            setFilters: () => buildURL({}, true),
+            loadSessions: () => buildURL({}, true),
             setSessionRecordingId: () => buildURL(),
             closeSessionPlayer: () => buildURL({ sessionRecordingId: undefined }),
```

Comment:
Just curious, why did you chose to make an exception here?

Reply:
You mean the last two? They basically go from the list to a individual recording, so that's basically going to a separate page that we could go back from.